examples/grpc_example/do_test

```diff
@@ -7,5 +7,5 @@ set -o errexit -o nounset -o pipefail
 bazel="${BIT_BAZEL_BINARY:-bazel}"
 
 # Ensure that it builds and tests pass
-"${bazel}" test //...
+"${bazel}" build //...
```

Comment:
Why did we switch away from test? We should at least try to run the thing that we just built.

Reply:
The GRPC authors removed the TestClient mode between 1.x and 2.x
Now the only option is just starting a server and communicating on localhost.
I did in different terminals run the client and server and verify the message round tripped, but I'm not sure how to integrate that with bazel testing.